test/OpenTelemetry.Exporter.Prometheus.Tests/PrometheusExporterExtensionsTests.cs

```diff
@@ -0,0 +1,77 @@
+// <copyright file="PrometheusExporterExtensionsTests.cs" company="OpenTelemetry Authors">
+// Copyright The OpenTelemetry Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.IO;
+using System.Text;
+using OpenTelemetry.Metrics;
+using OpenTelemetry.Tests;
+using Xunit;
+
+namespace OpenTelemetry.Exporter.Prometheus.Tests
+{
+    public sealed class PrometheusExporterExtensionsTests
+    {
+        [Fact]
+        public void WriteMetricsCollectionTest()
+        {
+            var tags = new KeyValuePair<string, object>[]
+            {
+                new KeyValuePair<string, object>("key1", "value1"),
+                new KeyValuePair<string, object>("key2", "value2"),
+            };
+
+            using var provider = Sdk.CreateMeterProviderBuilder()
+                .AddSource("TestMeter")
+                .AddReader(new BaseExportingMetricReader(new TestExporter<Metric>(RunTest)))
```

Comment:
Do we want to add a functional test, where Prometheus spins up a server/middleware, and the test make a webrequest to it, and validate the response contents?

Reply:
Definitely. My plan is to keep adding tests over multiple PRs until we reach some level of acceptable coverage.

Reply:
https://github.com/open-telemetry/opentelemetry-dotnet/pull/2030/files#diff-8c9aaed37ddc4021afc4e60bcdb8c381ebcd586c53808ea0634f2946c3173218 I had some basic one for that in the "old" metrics world. Not likely to be very useful, but just sharing.